_search-plugins/star-tree-index.md

```diff
@@ -7,77 +7,133 @@ nav_order: 54
 
 # Star-tree index
 
-This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
-{: .warning}
+A _star-tree index_ is a multi-field index that improves the performance of aggregations by precomputing metric values for combinations of dimension fields.
 
-A star-tree index is a multi-field index that improves the performance of aggregations.
+Once you enable star-tree indexes, OpenSearch automatically builds and uses star-tree indexes to optimize aggregations if the filter fields match the defined dimensions and the aggregation fields match the defined metrics in the star-tree configuration. No changes to your query syntax or request parameters are required.
```

Comment:
Star Tree is built during every Segment flush when enabled via field mapping on an index.

It is used for every Aggregation request if the filter fields match the defined dimensions and the aggregation fields match the defined metrics in the star-tree field mapping.

Reply:
@expani I'm just wondering if we need to call out the flush part, as a user anyways data is available during refresh / flush - so for them there is no difference in user experience

Reply:
@bharath-techie We are also talking about page cache efficiency at Line-18 which requires an intermediate/advanced user. So, thought we highlight a bit on the integration points with Lucene.